includes/polling/os/panos.inc.php

```diff
@@ -22,6 +22,81 @@
     $graphs['panos_sessions'] = true;
 }
 
+$sessions_tcp = snmp_get($device, '1.3.6.1.4.1.25461.2.1.2.3.4.0', '-Ovq');
```

Comment:
This would be much better if you could put all of these `snmp_get()` calls into a single `snmp_multi_get_oid()` call